src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -130,3 +135,10 @@ def load_command_table(self, _):
         g.command('list', 'list_authorization_rules')
         g.show_command('show', 'get_authorization_rule')
         g.command('keys list', 'list_keys')
+
+# MigrationConfigs Region
+    with self.command_group('servicebus migration', sb_migration_util, client_factory=migration_mgmt_client_factory) as g:
+        g.custom_command('start', 'cli_migration_start')
+        g.show_command('show', 'get', exception_handler=empty_on_404)
```

Comment:
remove the exception_handler. `show_command` will fail upon a 404 and exit with code of 3.
We do this to ensure consistency across all `show` commands.